docs/upgrade/2.8.0_to_2.9.0.rst

```diff
@@ -0,0 +1,134 @@
+.. _latest_upgrade_guide:
+
+Upgrade from 2.8.0 to 2.9.0
+===========================
+
+Update Servers to SecureDrop 2.9.0
+----------------------------------
+Servers running Ubuntu 20.04 will be updated to the latest version of SecureDrop
+automatically within 24 hours of the release.
+
+Update Workstations to SecureDrop 2.9.0 and Tails 6
+---------------------------------------------------
+If you have not already upgraded to Tails 6 alogside the 2.8.0 release,
+you should do so as part of this upgrade. Please note that the upgrade
+from Tails 6 must be performed manually.
```

Comment:
Maybe we should add "If you have already upgraded to Tails 6, you only need to complete Step 1 below."